node/src/testing/multi_stage_test_reactor.rs

```diff
@@ -123,17 +123,8 @@ impl MultiStageTestReactor {
         match self {
             MultiStageTestReactor::Deactivated => unreachable!(),
             MultiStageTestReactor::Initializer { .. } => None,
-            MultiStageTestReactor::Joiner { joiner_reactor, .. } => {
-                Some(joiner_reactor.consensus())
-            }
-            MultiStageTestReactor::JoinerFinalizing {
-                maybe_validator_init_config: None,
-                ..
-            } => None,
-            MultiStageTestReactor::JoinerFinalizing {
-                maybe_validator_init_config: Some(validator_init_config),
-                ..
-            } => Some(validator_init_config.consensus()),
+            MultiStageTestReactor::Joiner { .. } => None,
+            MultiStageTestReactor::JoinerFinalizing { .. } => None,
```

Comment:
Doesn't Clippy complain about identical match arms?

Reply:
Somehow it doesn't! But it might be prettier to merge them :+1: